core/shared/src/main/scala/monocle/function/Each.scala

```diff
@@ -25,12 +25,28 @@ trait EachFunctions {
 }
 
 object Each extends EachFunctions {
-  /** lift an instance of [[Each]] using an [[Iso]] */
-  def fromIso[S, A, B](iso: Iso[S, A])(implicit ev: Each[A, B]): Each[S, B] = new Each[S, B] {
-    val each: Traversal[S, B] =
-      iso composeTraversal ev.each
+
+  /** lift an instance of [[monocle.Traversal]] */
+  def apply[S, A](traversal: Traversal[S, A]) : Each[S, A] = new Each[S, A] {
+    override val each: Traversal[S, A] = traversal
   }
 
+  /** lift an instance of [[Each]] using an [[Iso]] */
+  def fromIso[S, A, B](iso: Iso[S, A])(implicit ev: Each[A, B]): Each[S, B] =
+    Each(iso composeTraversal ev.each)
+
+  /** lift an instance of [[monocle.Traversal]] using a [[Lens]] */
+  def fromLens[S, A, B](lens: Lens[S, A])(implicit ev: Each[A, B]): Each[S, B] =
```

Comment:
what the intended usage of the other `fromX`? 

`fromIso` was mainly done for newtypes

Reply:
you can but why not simply doing: `GenLens[RealUser](_.token) composeOptional possible`

Reply:
Yeah. Currently we have:
```scala
implicit val hasToken: Possible[RealUser, Token] = new Possible[RealUser, Token]  {
  val possible: Optional[RealUser, Token] = GenLens[RealUser](_.token) composePrism some
}
```
With the the `apply` helper method I could at least cut the boilerplate to:
```scala
  implicit val hasToken: Possible[RealUser, Token] = 
     Possible(GenLens[RealUser](_.token) composeOptional possible)
```
and yet a bit more with `fromLens`, but as you said I could live without it.

I can update this PR to only add the `apply` methods, if you prefer?

Reply:
yes I would prefer that we only add the apply method

Reply:
btw it might be good to do the same for all other typeclasses

Reply:
done!